js/modules/k6/browser/common/frame_manager.go

```diff
@@ -537,16 +529,13 @@ func (m *FrameManager) requestFinished(req *Request) {
 
 func (m *FrameManager) requestReceivedResponse(res *Response) {
 	m.logger.Debugf("FrameManager:requestReceivedResponse", "fmid:%d rurl:%s", m.ID(), res.URL())
-
-	m.page.emit(EventPageResponse, res)
```

Comment:
Do we still need this method if it only does logs?

Reply:
Agreed, removed in d94958f816a165533fa62631f114f3e23147c6bc